tests/performance_test/run_performance_test.py

```diff
@@ -6,91 +6,110 @@
 
```

Comment:
This file is fully rewritten in the improve performance test pull request.
You could skip the changes here to merge/rebase more easily.